src/quo/components/slideshow/slider_bar/schema.cljs

```diff
@@ -0,0 +1,13 @@
+(ns quo.components.slideshow.slider-bar.schema)
```

Comment:
@ajayesivan 
I'm agree about that small schemas and components could be in the same file. But is this approach already used somewhere? It looks like there are only discussions on this issue and there is no final decision yet

Reply:
Either way will be fine @Rende11 because nobody is strong about either option. When schemas get bigger they should probably go to a separate `schema.cljs` file, but this one is quite straightforward. But up to you :)